### PR TITLE
Tyler branch

### PR DESCRIPTION
**Pull request by Tyler-Pamperin**

### CODE REVIEW

File: academic-requirements/src/App.js [hunk 1]
```diff
@@ -11,14 +11,22 @@ function App() {
   const [major, setMajor] = useState('');
   const [concentration, setConcentration] = useState('');
 
+  /* 
+    Methods that assign major, minor, or concentration when picking option from a dropdown
+  */
+  function selectedMajor(_major) {
```
Comment (ruudkas) [inline academic-requirements/src/App.js:17]: Did you mean to include this?

File: academic-requirements/src/components/InputPage.tsx [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+  }
+
+  function selectedCourseNumber(_selectedNumber) {
+    setSelectedNumber(_selectedNumber);
+    // TODO update list of courses based on the selected course acronym and number
+
+  }
+  
+  // This method handles adding a new taken course to the table
+  function processCompletedCourse() {
```
Comment (ruudkas) [inline academic-requirements/src/components/InputPage.tsx:65]: This function is not up to date

**Review by ruudkas — CHANGES_REQUESTED** [review]
I don't think this code was supposed to be in the App file, it seems to work the same when I remove it, unless it has a future functionality that I am missing.
`    function selectedMajor(_major) {
      setMajor(_major);
      setShowConcentration(true);
    }`

There are quite a few changes missing from the input page file (validation). I am going to branch off of this branch and see if I can merge a few more in then create a pull request back into Tyler-Branch.

**Review by ruudkas — APPROVED** [review]
Looks good to me!

File: package.json [hunk 3]
```diff
@@ -1,5 +1,7 @@
 {
   "dependencies": {
-    "express": "^4.18.1"
+    "express": "^4.18.1",
+    "react": "^18.2.0",
```
Comment (ChrisVasterling) [inline package.json:4]: Is this react stuff supposed to be here? This is the package.json for the api.

Comment (Tyler-Pamperin) [reply package.json:4]: I know React Select is required for the dropdown to work. I'll try deleting them and seeing if anything breaks.

Comment (Tyler-Pamperin) [reply package.json]: Deleted them and it still seems to work as intended. I repushed with those deleted now. 👍

**Review by ChrisVasterling — APPROVED** [review]
Looks good to me

**Pull request merged**